app/components/Article/article.css

```diff
@@ -45,6 +45,7 @@ article .tag:hover {
 
 article .contents a {
   color: var(--colors-teal-500);
+  font-weight: bolder;
```

Comment:
No font properties should ever be specified in the CSS because we have type classes in newroot.css, always just apply one of those.

Can we only use font-weight: 400 or font-weight: 700?